commands/certmanager/post.go

```diff
@@ -0,0 +1,135 @@
+package certmanager
+
+import (
+	"context"
+	"fmt"
+	"os"
+
+	"github.com/ionos-cloud/ionosctl/pkg/core"
+	sdkgo "github.com/ionos-cloud/sdk-go-cert-manager"
+	"github.com/spf13/cobra"
+)
+
+var createProperties = sdkgo.CertificatePostPropertiesDto{}
+
+func CertPostCmd() *core.Command {
```

Comment:
same as `patch` command: this should be `create`, but the actual user-side functionality seems fine